src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.spec.ts

```diff
@@ -19,9 +19,8 @@ describe('TableComponent', () => {
     for (let i = 0; i < n; i++) {
       data.push({
         a: i,
-        b: i * i,
-        c: [-(i % 10), 'score' + ((i % 16) + 6)],
```

Comment:
I'm not sure why you removed it as it produces redundant data, which is needed if we want to match multiple items with one term and filter them out with another.